Lib/test/test_dataclasses.py

```diff
@@ -1022,19 +1022,25 @@ def test_class_var(self):
         class C:
             x: int
             y: int = 10
-            z: ClassVar[int] = 1000
-            w: ClassVar[int] = 2000
-            t: ClassVar[int] = 3000
-            s: ClassVar      = 4000
+            z: ClassVar[int]                    = 1000
+            w: ClassVar[int]                    = 2000
+            t: ClassVar[int]                    = 3000
+            s: ClassVar                         = 4000
+            a: Annotated[ClassVar, "meta"]      = 5000
```

Comment:
Could you add some tests with multiple levels of Annotated?

I'd also be interested in a stringified Annotated inside a non-stringified one.

Reply:
This is working, but just needs tests. I'll add some.